Core/Core/Planner/CalendarEvent/ViewModel/CalendarEventDetailsViewModel.swift

```diff
@@ -93,8 +93,8 @@ public final class CalendarEventDetailsViewModel: ObservableObject {
             .store(in: &subscriptions)
     }
 
-    private func onEventSetUpdateMenuButtonVisibility(userId: String) {
-        event
+    private func updateMenuButtonVisibility(onSet subject: CurrentValueSubject<CalendarEvent?, Never>, userId: String) {
```

Comment:
the argument `onSet` is quite confusing to me. It makes expect that some `Set` value is being passed to it. 
Can we rename to `on` as we have on other methods.

Reply:
The distinction I was trying to make here, was that `on: ` is to be used for actions (like `didTapButton`), while `onSet:` or `onChange:` is to be used for what are basically values. Strictly speaking both kind are just publishers, but one conveys an action, the other an assignment or change of a value.
With `onSet` instead of `onChange`, I tried to be explicit that the update happens for each assignment, not just for changes. But I'm not fully against using just `onChange` in these value cases.
I decided against `onSetOf` and `onChangeOf`, but maybe that would make it more clear.

Reply:
Yes, I think `onChange` would be more readable to me.

Reply:
Updating to `onChangeOf` in both methods